pkg/geo/geogfn/unary_operators.go

```diff
@@ -90,6 +93,41 @@ func Length(g *geo.Geography, useSphereOrSpheroid UseSphereOrSpheroid) (float64,
 	return length(regions, useSphereOrSpheroid)
 }
 
+// Project returns calculate a projected point given a source point, a distance and a azimuth.
+func Project(point *geom.Point, distance float64, azimuth float64) (*geom.Point, error) {
+	spheroid := geographiclib.WGS84Spheroid
+
+	// Normalize distance to be positive.
+	if distance < 0.0 {
+		distance = -distance
+		azimuth += math.Pi
+	}
+
+	// Normalize azimuth
+	azimuth -= 2.0 * math.Pi * math.Floor(azimuth/(2.0*math.Pi))
```

Comment:
if we are using `s1.Angle()`, there is a `Normalized` function we can use.

Reply:
Done.